lib/utils/receipt.js

```diff
@@ -12,9 +12,12 @@ function Receipt(tx, block, logs, gasUsed, cumulativeGasUsed, contractAddress, s
 
   this.transactionIndex = 0;
 
+  let inclSig = tx.constructor.name !== "Transaction";
```

Comment:
I'm looking into add _isRawTx everywhere now.  In both cases I was using the constructor.name to avoid an erroneous Transaction `require`.  Im totally onboard using instanceof, especially if its not a performance hit on the extra require / is more robust than using constructor.name.